TheChocolateRiversDB/3 table.sql

```diff
@@ -0,0 +1,38 @@
+go
+use ChocolateRiverDB
+go
+drop table if exists dbo.Chocolate
+go
+create table dbo.Chocolate(
+    ChocolateID int not null identity primary key, 
+ChocolateType int not null 
+  constraint ck_Chocolate_chocolate_type_must_be_1_2_or_3 check (ChocolateType between 1 and 3),
+ChocolateTypeDesc as case ChocolateType
+when 1 then 'Solid Block'
+when 2 then 'Chocolate Trouffles'
+when 3 then 'Chocolate Candy Bar'
+end,
+Flavor int not null
+  constraint ck_Chocolate_flavor_must_be_1_through_5 check (Flavor between 1 and 5),
+FlavorDesc as case Flavor
+when 1 then 'Caramalized white chocolate'
+when 2 then 'Milk Chocolate Hazelnut Esspresso'
+when 3 then 'Dark 54 Chocolate Rasberry Pomegranate'
+when 4 then 'Dark salted caramel'
+when 5 then 'milk creme & Dark Creme'
+end,
+Shape varchar(7) not null
+constraint ck_Chocolate_Shape_must_be_heart_or_oval_or_square_or_oracle_or_ripple_or_thimble check (Shape in ('heart','oval','square','oracle','ripple','thimble')),
+ChocolateName as concat(Shape, ' Shaped ', ChocolateType),
```

Comment:
-- LL the chocolate name should include the flavor before the chocolate type and end in a period

Reply:
--change columns to just text description

Reply:
@leahlebovits - Hi,
I realized that removing the numerical descriptions won't solve this problem unless I insert the Chocolate Type and Flavor in the data, as apposed to a computed column.